opa-manager-operator/testing-yamls/template-constraint.yaml

```diff
@@ -0,0 +1,20 @@
+ 
```

Comment:
Where is this file used?

Reply:
these are example files, maybe I should not have commited. Will remove from next commit